howso/direct/core.py

```diff
@@ -757,6 +757,78 @@ def get_num_training_cases(self, trainee_id: str) -> Dict:
         """
         return self._execute("get_num_training_cases", {"trainee": trainee_id})
 
+    def get_auto_ablate_params(self, trainee_id: str):
+        """
+        Get parameters set by :meth:`set_auto_ablate_params`.
```

Comment:
Do we wanna match this more with `set_auto_ablate_params`?

something more like "Get trainee parameters for auto ablation."